WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController+Header.swift

```diff
@@ -77,6 +78,7 @@ extension BlogDetailsViewController {
                                 feedbackType: .success)
             ActionDispatcher.global.dispatch(NoticeAction.post(notice))
 
+            self?.headerView.updateLoadingTitle(isLoading: false)
             self?.headerView.refreshSiteTitle()
         }, failure: { [weak self] error in
             self?.blog.settings?.name = existingBlogTitle
```

Comment:
The header view is currently not updated if there's a failure with the request to update the settings, which results it showing the loading indicator indefinitely. We should also stop loading here and restore the original title.

Reply:
Absolutely! Thank you!

Reply:
Done in 1e49f70d7941a7b87ab0c07795ec71643224fcac